src/components/Buttons/PlusButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactElement } from "react";
+import cn from "classnames";
+import Button from "../Button";
+
+const SMALL = 'small';
+const MEDIUM = 'medium';
+
+type Props = {
+  className: string;
```

Comment:
Same here :) `className` can be optional and I guess we can default the `size` to `medium` and make it optional as well.

Reply:
ok, but className is already set on "medium", it is declared as default props at the end of the code :) I'll just make optional className and type

Reply:
👍